config/webhook/service.yaml

```diff
@@ -4,10 +4,6 @@ metadata:
   name: webhook-service
   namespace: system
 spec:
-  ports:
```

Comment:
why is this removed?

Reply:
To avoid having duplicate ports definition in Service specs. 
Instead of
```
  ports:
   - name: https-webhook-server
     port: 443
     targetPort: webhook-server
```

will be
```
  {{- if .Values.service.ports }}
   ports:
   - name: https-webhook-server
     port: {{ .Values.service.ports.https }}
     targetPort: webhook-server
   - name: metrics
     port: {{ .Values.service.ports.metrics }}
     protocol: TCP
     targetPort: metrics
   {{- end }}
```

and after apply default values
```
   ports:
   - name: https-webhook-server
     port: 443
     targetPort: webhook-server
   - name: metrics
     port: 8888
     protocol: TCP
     targetPort: metrics
```